packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -77,17 +78,97 @@ enum SystemUiOverlay {
   bottom,
 }
 
+/// Specifies a dark vs light preference for certain system chrome options.
+enum SystemChromeTheme {
+  /// Whether the setting should be drawn to be compatible with a light color.
+  light,
+  /// Whether the setting should be drawn to be compatible with a dark color.
+  dark,
+}
+
 /// Specifies a preference for the style of the system overlays.
 ///
 /// Used by [SystemChrome.setSystemUIOverlayStyle].
-enum SystemUiOverlayStyle {
+class SystemUiOverlayStyle {
   /// System overlays should be drawn with a light color. Intended for
   /// applications with a dark background.
-  light,
+  static const SystemUiOverlayStyle light = const SystemUiOverlayStyle(
+    navigationBarColor: 0xFF000000,
```

Comment:
Black seems at odds with `overlays should be drawn with a light color`

Reply:
Let me do another pass on these values... I keep getting my wires crossed whether we a referring to color of the UI or the color to match the UI.